pkg/aws/testutil_test.go

```diff
@@ -102,10 +135,10 @@ func fakeResourceTagMappingTag() *resourcegroupstaggingapi.Tag {
 	}
 }
 
-func fakeResourceTagMapping() *resourcegroupstaggingapi.ResourceTagMapping {
+func fakeResourceTagMapping(arn string) *resourcegroupstaggingapi.ResourceTagMapping {
```

Comment:
Should we be passing in a `modifyFn` here to keep this a bit more dynamic?
```suggestion
func fakeResourceTagMapping(modifyFn func(*resourcegroupstaggingapi.ResourceTagMapping)) *resourcegroupstaggingapi.ResourceTagMapping {
	mock := &resourcegroupstaggingapi.ResourceTagMapping{
		ComplianceDetails: nil,
		ResourceARN:       aws.String(fakeResourceTagMappingARN),
		Tags: []*resourcegroupstaggingapi.Tag{
			fakeResourceTagMappingTag(),
		}
	if modifyFn != nil {
		modifyFn(mock)
	}
	return mock
}
```